src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-daemon-list/rgw-daemon-list.component.html

```diff
@@ -25,7 +25,7 @@
     <ng-template ngbNavContent>
       <cd-grafana [grafanaPath]="'rgw-overview?'"
                   uid="WAkugZpiz"
-                  grafanaStyle="two">
+                  grafanaStyle="three">
```

Comment:
```suggestion
                  grafanaStyle="two">
```

Reply:
isn't `two` enough?
You changed the other iframe, which I was not able to test.